### ISSUE TITLE
mintty keeps freezing for a few seconds

### ISSUE THREAD

**Issue by jk3us** [mintty/mintty#771]
The last week or so, mintty (for cygwin) will freeze up for several seconds (typically around 10 seconds), then work normally again for a while.  This is mintty 2.8.5 (i686-pc-cygwin).  All of my cygwin packages are up to date. What can I do to try to figure out what is happening before or during those times?

**Comment by mintty** [comment]
First, please go back to mintty 2.8.4 (select previous version in cygwin setup) and check whether it still happens.

**Comment by jk3us** [comment]
So, I downgraded to 2.8.4 and worked for a while without encountering this problem. Strangely, that also seemed to fix a different problem I was having that I thought couldn't possibly be related.  Some git commands would return with no error, but wouldn't actually do anything.  I was having that problem even if I ran cygwin under `cmd`.

So after that, I upgraded to mintty 2.8.5 again to see if either of the problems would resurface, and so far neither has (the git problem is still fixed).

So, unless the problem resurfaces again, I guess we can put this in the "inexplicably works now" bin.
